src/exodus/walletdb.h

```diff
@@ -0,0 +1,129 @@
+#ifndef EXODUS_WALLET_DB_H
+#define EXODUS_WALLET_DB_H
+
+#include "wallet/walletdb.h"
+#include "secp256k1/include/GroupElement.h"
+#include "serialize.h"
+
+#include "sigma.h"
+
+#include <string>
+
+namespace exodus {
+
+class SigmaMintId
+{
+public:
+    SigmaMintId()
+        : commitment(GroupElement()), propertyId(0), denomination(0)
+    {
+    }
+    SigmaMintId(const secp_primitives::GroupElement& commitment, uint32_t propertyId, uint8_t denomination)
+        : commitment(commitment), propertyId(propertyId), denomination(denomination)
+    {
+    }
+
+    secp_primitives::GroupElement commitment;
```

Comment:
We should change to `SigmaPublicKey` instead of `secp_primitives::GroupElement`.